src/cmd/build.js

```diff
@@ -60,8 +61,21 @@ function defaultPackageCreator(
           filter: (...args) => fileFilter.wantFile(...args),
         })
         .then((buffer) => {
+          let messageData: any;
+          if (manifestData.default_locale) {
+            messageData = readFileSync(
+               path.join(sourceDir, '_locales', manifestData.default_locale,
+                'messages.json'));
+          }
+          let extensionName: string;
+          if (messageData) {
+            extensionName = JSON.parse(messageData).extensionName.description;
```

Comment:
@kumar303 also wouldn’t it be better to localize the `extensionName` directly when generating the [manifestData](https://github.com/mozilla/web-ext/blob/2c06dbd892feb02bd69a0345bee2b4d2d98233b1/src/util/manifest.js) instead of during build and possibly in other places in the future?